builtin/game/falling.lua

```diff
@@ -6,6 +6,33 @@ local builtin_shared = ...
 -- Falling stuff
 --
 
+local node_fall_hurt = minetest.setting_getbool("node_fall_hurt") ~= false
+
+local function fall_hurt_check(self, pos, dtime)
+	if not node_fall_hurt then return end
+	if self.hurt_timer > 0 then
+		self.hurt_timer = self.hurt_timer - dtime
+	else
+		-- Get damage level from falling_node_damage group
+		local damage = core.registered_nodes[self.node.name] and
```

Comment:
Ok i'm not too bothered, but you can cache `core.registered_nodes[self.node.name]` in a variable, as Krock mentioned on IRC.